packages/@react-spectrum/searchwithin/test/SearchWithin.test.js

```diff
@@ -53,9 +53,9 @@ describe('SearchWithin', function () {
   });
 
   it('renders correctly', function () {
-    let {getAllByText, getByRole, getByLabelText} = renderSearchWithin();
+    let {getAllByText, getByRole, getByPlaceholderText} = renderSearchWithin();
```

Comment:
out of curiosity, why were these all moved to getByPlaceholderText? i think eventually we want to get rid of placeholder text, so it's more future proof if we get it by the label text, if possible

Reply:
That query was failing for some reason after a change. So I decided to use the placeholder one since it would work for those and also the ones without a label. I think you're right though, I'll see if I can replace that.

Reply:
`getByRole('searchbox')` works, so I'll use that instead. Thanks!